src/utils/checkClassesMethods.ts

```diff
@@ -0,0 +1,7 @@
+export const checkClassesMethods = (classes: any) => {
```

Comment:
```suggestion
export function getNoMethodClasses(classes: any) {
```

I find this more readable.